bot/bot.py

```diff
@@ -13,6 +15,13 @@ def __init__(self):
 
             self.bot = AzureChatGPTBot()
         else:
+            model = conf().get("model", "gpt-3.5-turbo")
+            if model in litellm.model_list:
+                # see litellm supported models here:
+                # https://litellm.readthedocs.io/en/latest/supported/
+                from bot.litellm import liteLLMChatGPTBot
+                self.bot = liteLLMChatGPTBot()
+
             from bot.chatgpt import ChatGPTBot
```

Comment:
The self.bot will always be assigned by ChatGPTBot instance, seems there needs to be  an else block here.